usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -103,12 +103,15 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/search/spending_by_category/awarding_agency/](/api/v2/search/spending_by_category/awarding_agency/)|POST| Returns data that is grouped in preset units to support the Spending by Awarding Agency data visualizations on USAspending.gov's Advanced Search page|
 |[/api/v2/search/spending_by_category/awarding_subagency/](/api/v2/search/spending_by_category/awarding_subagency/)|POST| Returns data that is grouped in preset units to support the Spending by Awarding Subgency data visualizations on USAspending.gov's Advanced Search page|
 |[/api/v2/search/spending_by_category/cfda/](/api/v2/search/spending_by_category/cfda/)|POST| Returns data that is grouped in preset units to support the Spending by CFDA data visualizations on USAspending.gov's Advanced Search page|
+|[/api/v2/search/spending_by_category/district/](/api/v2/search/spending_by_category/district/)|POST| Returns data that is grouped in preset units to support the Spending by Congressional District data visualizations on USAspending.gov's State Profile page|
```

Comment:
not quite in alphabetical order, if that was the intention.

Reply:
Definitely the intention...and definitely overlooked.